go/obscuronode/host/hostrunner/cli.go

```diff
@@ -57,6 +58,7 @@ func ParseConfig() config.HostConfig {
 	logPath := flag.String(logPathName, defaultConfig.LogPath, logPathUsage)
 	chainID := flag.Int64(chainIDName, defaultConfig.ChainID.Int64(), chainIDUsage)
 	privateKeyStr := flag.String(privateKeyName, defaultConfig.PrivateKeyString, privateKeyUsage)
+	contractMgmtBlkHash := flag.String(contractMgmtBlkHashName, common.Hash{}.Hex(), contractMgmtBlkHashUsage)
```

Comment:
For consistency, it would be good to use `defaultConfig.ContractMgmtBlkHash` here.

Reply:
👍